### PR TITLE
Software 0.25.15 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: software_versioned_docs/version-0.25/release-notes.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ For more Astronomer Software release notes, see:
...
 All Astronomer Software versions are tested for scale, reliability and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
 
+## 0.25.15
+
+Release date: September 21, 2022
+
+### Bug fixes 
+
+- Fixed the following vulnerabilities:
+    - [CVE-2022-40674](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-40674)
```
Comment (jwitz) [inline software_versioned_docs/version-0.25/release-notes.md:30]: @vishwas-astro Do we have a list of the CVEs we managed to resolve from this original list? https://github.com/astronomer/issues/issues/5047

File: software_versioned_docs/version-0.25/release-notes.md [hunk 2]
```diff
@@ -20,6 +20,15 @@ For more Astronomer Software release notes, see:
 
 All Astronomer Software versions are tested for scale, reliability and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
 
+## 0.25.15
+
+Release date: September 21, 2022
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.25/release-notes.md:25]:
```suggestion
Release date: September 21, 2022
```
Is this date correct?

File: software_versioned_docs/version-0.25/release-notes.md [hunk 3]
```diff
@@ -20,6 +20,15 @@ For more Astronomer Software release notes, see:
 
 All Astronomer Software versions are tested for scale, reliability and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.25/release-notes.md:21]:
```suggestion
All Astronomer Software versions are tested for scale, reliability, and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
```

**Review by kwcanuck — APPROVED** [review]

File: software_versioned_docs/version-0.25/release-notes.md [hunk 4]
```diff
@@ -20,6 +20,15 @@ For more Astronomer Software release notes, see:
...
 
+## 0.25.15
+
+Release date: October 14, 2022
```
Comment (vishwas-astro) [inline software_versioned_docs/version-0.25/release-notes.md:25]: 0.25.15 was just released today. Can you update the date to Oct 18th?

File: software_versioned_docs/version-0.25/release-notes.md (hunk 1 above)
Comment (vishwas-astro) [reply software_versioned_docs/version-0.25/release-notes.md:30]: CVE-2021-44906 - https://nvd.nist.gov/vuln/detail/CVE-2021-44906
CVE-2022-32213 - https://nvd.nist.gov/vuln/detail/CVE-2022-32213
CVE-2022-32214 - https://nvd.nist.gov/vuln/detail/CVE-2022-32214
CVE-2022-32215 - https://nvd.nist.gov/vuln/detail/CVE-2022-32215
CVE-2022-24785 - https://nvd.nist.gov/vuln/detail/CVE-2022-24785
CVE-2022-31129 - https://nvd.nist.gov/vuln/detail/CVE-2022-31129
CVE-2022-25878 - https://nvd.nist.gov/vuln/detail/CVE-2022-25878

**Pull request merged**